include/sentry.h

```diff
@@ -1890,12 +1890,44 @@ SENTRY_EXPERIMENTAL_API void sentry_options_set_enable_logs(
 SENTRY_EXPERIMENTAL_API int sentry_options_get_enable_logs(
     const sentry_options_t *opts);
 
+/**
+ * The potential returns of calling any of the sentry_logs_X functions
+ * - Success means a log was enqueued
+ * - Discard means the `before_send_log` function discarded the log
+ * - Failed means the log wasn't enqueued. This happens if the buffers are full
+ * - Disabled means the option `enable_logs` was false.
+ */
 typedef enum {
     SENTRY_LOG_RETURN_SUCCESS = 0,
     SENTRY_LOG_RETURN_DISCARD = 1,
     SENTRY_LOG_RETURN_FAILED = 2,
     SENTRY_LOG_RETURN_DISABLED = 3
 } log_return_value_t;
+
+/**
+ * Structured logging interface. Minimally blocks the client trying to log,
+ * but is therefor lossy when enqueueing a log fails
```

Comment:
I think this is a classic example of "therefore/therefor" as a conjunctive adverb meaning: "we do lossy, because we don't want to block you" and it that case it is the one with an "e" at the end:
```suggestion
 * but is therefore lossy when enqueueing a log fails
```